lhotse/cut.py

```diff
@@ -489,6 +490,7 @@ def index_supervisions(
                     )
         return indexed
 
+    @deprecated("Please use save_audio() instead.")
```

Comment:
how about sth like: 

```suggestion
    @deprecated("Cut.compute_and_store_recording will be removed in a future release. Please use save_audio() instead.")
```

Reply:
also to avoid unnecessary code duplication this function could just call `save_audio` under the hood.